### PR TITLE
Changes about electronic give out for SIMCards.

### PR DESCRIPTION
**Pull request by lukaszkarykowski**
Added modules from BackOffice transitions to SIMCards module.
This modules allows to accept give out of SIMCards electronically.

### CODE REVIEW

File: src/ralph/admin/templatetags/dashboard_tags.py [hunk 1]
```diff
@@ -54,6 +56,18 @@ def get_user_equipment_to_accept_tile_data(user):
     }
 
 
+def get_user_simcard_to_accept_tile_data(user):
```
Comment (ar4s) [inline src/ralph/admin/templatetags/dashboard_tags.py:59]: Could you provide some test for this function? Hint: mock ``get_simcards_to_accept``to simplify testing.

File: src/ralph/sim_cards/models.py [hunk 2]
```diff
@@ -157,6 +165,101 @@ def assign_user(cls, instances, **kwargs):
...
+                    )
+                )
+            },
+        }
+    )
+    def accept_asset_release_agreement(cls, instances, requester, **kwargs):
+        pass
+
+    @classmethod
+    def _generate_report(cls, name, requester, instances, language):
```
Comment (ar4s) [inline src/ralph/sim_cards/models.py]: DRY - extract `_generate_report` to common function and use it in `back_office/model.py` and here.

**Review by ar4s — CHANGES_REQUESTED** [review]

File: src/ralph/admin/templatetags/dashboard_tags.py (hunk 1 above)
Comment (lukaszkarykowski) [reply src/ralph/admin/templatetags/dashboard_tags.py:59]: Test passed, and pushed to git.

File: src/ralph/sim_cards/models.py (hunk 2 above)
Comment (lukaszkarykowski) [reply src/ralph/sim_cards/models.py]: Changed. And I push with changes.

**Review by ar4s — APPROVED** [review]

**Review by szymi- — APPROVED** [review]

**Review by mkurek — APPROVED** [review]

**Pull request merged**